recipes/magnum/all/test_package/conanfile.py

```diff
@@ -42,3 +43,7 @@ def test(self):
 
             bin_path = os.path.join("bin", "test_package")
             self.run(bin_path, run_environment=True)
+
+        if self.settings.os == "Emscripten":
+            bin_path = os.path.join("bin", "test_package.js")
+            self.run("node {}".format(bin_path), run_environment=True)
```

Comment:
are you adding `nodejs` as a requirement somewhere?
as `emsdk` is build require, will it disappear from graph as soon as `magnum` is built?

Reply:
I think it is fine: `nodejs` is a requirement of `emsdk`, if you are building for `os=Emscripten` then `emsdk` is already a build-requires (or it is in the environment) and `node` should be available in PATH

Reply:
okay, I think it's good